### PR TITLE
Added support for Microsoft Stream

### PR DESCRIPTION
**Pull request by michaelkamprath**
The v0.3.0 release dropped support for Microsoft Stream. This change adds it back in.

### CODE REVIEW

**Comment by michaelkamprath** [comment]
@hoveeman Since you originally implemented Microsoft Stream support, and since I am not in a position to test whether this change works, could you review this pull request?

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ All notable changes to this project will be documented in this file.
 
 ### Fixed
 * Correct bug in RTMP configuration generation when the audio bit rate is set.
+* Reinstated support for Microsoft Stream
+<<<<<<< HEAD
```
Comment (michaelkamprath) [inline CHANGELOG.md:11]: Huh?

Comment (michaelkamprath) [reply CHANGELOG.md]: OK, fixed with 2c1bbe23a6c9eeebc7b6e1d4791da0797c088abb

**Comment by hoveeman** [comment]
Great! I’ll try and take a look at it this weekend and test it out. Appreciate you keeping Microsoft Stream in the mix.

**Comment by hoveeman** [comment]
Worked perfectly! Great job! I was also streaming to YouTube, Facebook, and Periscope at the same time to make sure they all worked together.

Screen Shot 2020-05-09 at 2 25 19 PM

**Comment by michaelkamprath** [comment]
@hoveeman great, thanks!

**Pull request merged**